dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -104,3 +104,7 @@ enableImplicitBeanArchives.conflict.CWOWB1017W.useraction=Update your server.xml
 unversioned.beansXML.warning.CWOWB1018W=CWOWB1018W: Using an unversioned CDI beans.xml file might lead to unexpected behaviour when using CDI 4.0 or later: {0}
 unversioned.beansXML.warning.CWOWB1018W.explanation=Prior to CDI 4.0, a non-empty but unversioned beans.xml file meant that the archive was treated as an explicit bean archive (all classes are eligible to be beans). In CDI 4.0 or later, the archive is taken to be an implicit bean archive (only annotated classes are eligible be beans).
 unversioned.beansXML.warning.CWOWB1018W.useraction=Update the beans.xml file within your application to use a newer version of the schema and ensure that the version and bean-discovery-mode attributes are properly set.
+
+no.bda.for.module.CWOWB1019E=CWOWB1019E: There was no bean definition archive for module {0}
+no.bda.for.module.CWOWB1019E.explanation=Every module in a CDI enabled application must have a bean definition archive to store CDI's knowledge of that modules contests. However a module was missing a bean definition archive.
+no.bda.for.module.CWOWB1019E.useraction=Review logs to identify problems in application startup which may cause this issue.
```

Comment:
This message talks about things that we don't expose to users. We should probably just use a generic "A CDI internal error occurred while doing X" type error message, with the exception appended.